packages/@okta/i18n/src/properties/login.properties

```
@@ -1023,6 +1023,11 @@ securityQuestion.answer.tooShort.arg = The security question answer must be at l
 oie.okta_verify.label = Okta Verify
 oie.okta_verify.authenticator.description = Okta Verify is an authenticator app, installed on your phone, used to prove your identity
 oie.enroll.okta_verify.setup.title = Set up Okta Verify
+oie.enroll.okta_verify.setup.subtitle = To continue, you'll need an Okta Verify account on this device.
+oie.enroll.okta_verify.setup.download = If you don't have Okta Verify installed, <a href="{0}" target="_blank" class="download-ov-link" id="download-ov">download the app</a>.
```

We try to avoid HTML embedded in i18n strings, can you please modify this to mimic [this](https://github.com/okta/okta-signin-widget/blob/master/packages/%40okta/i18n/src/properties/login.properties#L1134) approach?

```suggestion
oie.enroll.okta_verify.setup.download = If you don't have Okta Verify installed, <$1>download the app</$1>.
```

---

I know the implementation is not apart of this PR, but just for future reference, when it is used, it will be used like [this](https://github.com/okta/okta-signin-widget/blob/master/src/v2/view-builder/views/ov/ResendNumberChallengeView.js)